src/core/server/server.api.md

```diff
@@ -1790,6 +1790,10 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
     authRequired?: boolean | 'optional';
     body?: Method extends 'get' | 'options' ? undefined : RouteConfigOptionsBody;
     tags?: readonly string[];
+    timeout?: {
+        server?: false | number;
+        socket?: false | number;
```

Comment:
Will it be removed once https://github.com/elastic/kibana/pull/73407 merged?

Reply:
Yep, these docs will change once that is merged.